hack/install-vm.sh

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env bash
+​
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+​
+INSTALL_PATH="/usr/sbin"
+ANTREA_AGENT="antrea-agent"
+AGENT_LOG_DIR="/var/log/antrea"
+AGENT_CONF_PATH="/etc/antrea"
+OVS_BRIDGE="br-int"
+OVS_VSWITCHD="ovs-vswitchd.service"
+ANTREA_DOCKER_IMAGE="projects.registry.vmware.com/antrea/antrea-ubuntu:v1.9.0"
+OVS_BIN="start_ovs"
+ANTREA_AGENT_BIN="antrea-agent"
+ANTREA_AGENT_KUBECONFIG="antrea-agent.kubeconfig"
+ANTREA_AGENT_ANTREA_KUBECONFIG="antrea-agent.antrea.kubeconfig"
+
+_usage="Usage: $0 [--ns <Namespace>] [--bin <AntreaAgentSavePath>] [--config <AgentConfigSavePath>] [--kubeconfig <KubeconfigSavePath>] [--antrea-kubeconfig <AntreaKubeconfigSavePath>] [--nodename <ExternalNodeName>] [--ovs-bridge <OVSBridgeName>] [--install-docker <InstallDocker>] [--validate-ovs] [--help|-h]
+        --ns                          Namespace to be used by the antrea-agent
+        --bin                         Path of the antrea-agent binary
+        --config                      Path of the antrea-agent configuration file
+        --kubeconfig                  Path of the kubeconfig to access K8s API Server
+        --antrea-kubeconfig           Path of the kubeconfig to access Antrea API Server
+        --nodename                    ExternalNode name to be used by the antrea-agent
+        --ovs-bridge                  Specify the OVS bridge name
+        --validate-ovs                Validate OVS configuration and performs cleanup when any error is detected.
+        --install-docker              Specify true or false to install or skip the docker installations.
```

Comment:
```suggestion
        --dockerize             To run antrea-agent in a docker container
```

Also no need to pass the value to this argument. If this option is specified, we should install and run antrea in docker container.